ui/src/app/applications/components/application-status-panel/application-status-panel.scss

```diff
@@ -11,7 +11,7 @@
     box-shadow: 1px 2px 3px rgba(0, 0, 0, 0.1);
 
     &__item {
-        height: 75px;
+        max-height: 100px;
```

Comment:
I think this should still be set at 75px. For shorter errors, setting at 100px leaves an awkward gap: 

<img width="485" alt="Screen Shot 2021-01-14 at 11 02 40 AM" src="https://user-images.githubusercontent.com/6250584/104636319-08d1a580-5658-11eb-97a5-a26f85e471f8.png">